tests/unit/EntityName.spec.ts

```diff
@@ -355,17 +355,17 @@ describe('Name Changes for a SP alteration', () => {
     // Verify the conflict flag is true when the NR legal type is different than the current business type
     expect(wrapper.vm.isConflictingLegalType).toBe(true)
 
-    expect(companyInfo.at(0).text()).toBe('Business Type:  BC Company')
-    expect(companyInfo.at(1).text()).toBe('Request Type:  New Business')
-    expect(companyInfo.at(2).text()).toBe('Expiry Date:  March 10, 2021 at 12:00 am Pacific time')
-    expect(companyInfo.at(3).text()).toBe('Status:  approved')
+    expect(companyInfo.at(0).text()).toBe('Business Type:')
```

Comment:
Should this check for "BC Company" like before?

Reply:
Please check last commit Sev. The test was setting wrong the `legalType` wrongly. It should be fine now.